src/redux/optimismExplorer.js

```diff
@@ -83,30 +82,28 @@ export const optimismExplorerInit = () => async (dispatch, getState) => {
       return;
     }
 
-    const tokenAddresses = assets.map(
-      ({ asset: { asset_code } }) => asset_code
+    const tokenAddresses = assets.map(({ asset: { asset_code } }) =>
+      toLower(asset_code)
```

Comment:
the `toLower` will guarantee that the balances key can be looked up accurately later